support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -40,27 +40,37 @@ export type StripePaymentMethod = 'StripeCheckout' | 'StripeApplePay' | 'StripeP
 export type StripePaymentRequestButtonMethod = 'none' | StripePaymentMethod;
 
 type RegularContribution = {|
+  productType: 'Contribution',
   amount: number,
   currency: string,
   billingPeriod: BillingPeriod,
 |};
 
 export type DigitalSubscription = {|
+  productType: 'DigitalSubscription',
   currency: string,
   billingPeriod: BillingPeriod,
   readerType: ReaderType,
-  giftMessage?: string | null,
-  giftDeliveryDate?: string | null,
```

Comment:
these are not used on the server side - must have been left over from an earlier iteration

Reply:
How do those fields get to the server then?

Reply:
they are in the giftRecipient field instead

Reply:
subscription created in UAT during testing has the delivery date correctly populated https://apisandbox.zuora.com/apps/Subscription.do?method=view&id=2c92c0f87628864d017628d7c7185530